docs/_tabsets/install.qmd

```diff
@@ -54,7 +54,7 @@ for installer in installers:
         print()
         print(f"```bash\n{cmd.format(extra=extra)}\n```")
         print()
-        print(f"Connect using [`ibis.{mod}.connect`](../backends/{name.lower()}.qmd#ibis.backends.{mod}.Backend.do_connect).")
+        print(f"Connect using [`ibis.{mod}.connect`](/backends/{name.lower()}#ibis.backends.{mod}.Backend.do_connect).")
```

Comment:
Tried this locally, it didn't work 😕 

I just tried this locally and it fixes the links and doesn't seem to break other references (e.g., `docs/posts/v6.1.0-release/index.qmd`):

```suggestion
        print(f"Connect using [`ibis.{mod}.connect`](./backends/{name.lower()}.qmd#ibis.backends.{mod}.Backend.do_connect).")
```